backend/README.md

```diff
@@ -14,4 +14,26 @@ To unit test the backend, run the following command in the backend folder:
 
 ```
 dotnet test
-```
\ No newline at end of file
+```
+
+## Formatting
+
+The formatting of the backend is defined in the [.editorconfig file](../editorconfig).  
+
+We use [dotnet format](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-format)
+to format and verify code style in backend based on the 
+[C# coding conventions](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions).  
+
+Dotnet format is included in the .NET6 SDK.
+
+To check the formating, run 
+```
+cd backend
```

Comment:
dont we need to be in the backend/api folder?

Reply:
This also goes for the `dotnet run` command above and for test you need to be in backend/test no?

Reply:
No, the solution file gives dotnet the info it needs, so we can run both commands from the backend folder

Reply:
Ah, that doesnt work for VSCode (maybe I need an extension or something), but Im fine with that

Reply:
It fails to run the command in the backend directory? I thought the solution file would give the information even outside visual studio